pkgs/test_core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 0.5.7-wip
 
+* Update the CompilerPool to be generic, so that custom compiler pools can
```

Comment:
This isn't breaking?

Reply:
It can default to dynamic safely right? The return types were always just `Future<void>` before.